api/types/Thread.js

```diff
@@ -1,5 +1,14 @@
 // @flow
 const Thread = /* GraphQL */ `
+  enum ThreadReactionTypes {
+		like
+	}
+
+  type ThreadReactions {
+		count: Int!
+		hasReacted: Boolean
+	}
```

Comment:
Even though you split 'em up, this is still the same schema as the message reactions? I thought we agreed that we should do this one right from the start (to one that let's us show who liked a thread) so we wouldn't have to migrate to a different one in the future?

Reply:
Right, see comment below.